.github/workflows/kvrocks.yaml

```diff
@@ -35,7 +35,7 @@ concurrency:
 jobs:
   check-and-lint:
     name: Lint and check code
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
```

Comment:
We need newer clang-tidy (version 12 -> 14) so that a bug (false report) in old version will be fixed.